backends/magma/ceed-magma-basis.c

```diff
@@ -372,6 +372,22 @@ int CeedBasisCreateTensorH1_Magma(CeedInt dim, CeedInt P1d, CeedInt Q1d,
   Ceed ceed;
   ierr = CeedBasisGetCeed(basis, &ceed); CeedChk(ierr);
 
+  // Check for supported parameters
```

Comment:
@nbeams feel free to modify, I just added this to move the error message up to basis creation rather than application

Reply:
LGTM, thanks